docs/EN_US/ECLWatch/ECLWa_mods/ECLWatchSrc.xml

```diff
@@ -1387,6 +1387,55 @@
           <para>Press the <emphasis role="bold">Copy</emphasis> button to
           display the copy drop menu, with additional file copy
           options.</para>
+
+          <!--Now this here would be a really good spot to XInclude a dynamically built options table-->
+
+          <para><itemizedlist>
+              <listitem>
+                <para>Check the<emphasis role="bold"> Overwrite</emphasis> box
+                to overwrite files of the same name.</para>
+              </listitem>
+
+              <listitem>
+                <para>Check the<emphasis role="bold"> Compress</emphasis> box
+                to compress the file copy.</para>
+              </listitem>
+
+              <listitem>
+                <para>Check the<emphasis role="bold"> Retain Superfile
+                Structure</emphasis> box for the copy to retain the structure
+                of a Superfile.</para>
+              </listitem>
+
+              <listitem>
+                <para>Check the<emphasis role="bold"> Replicate</emphasis> box
+                to create backup copies of all file parts.</para>
+              </listitem>
+
+              <listitem>
+                <para>Check the<emphasis role="bold"> No Split</emphasis> box
+                to to prevent splitting up the file copy into parts.</para>
+              </listitem>
+
+              <listitem>
+                <para>Check the<emphasis role="bold"> Wrap</emphasis> box to
+                keep the number of parts the same and wrap if the target
+                cluster is smaller than the original.</para>
+              </listitem>
+
+              <listitem>
+                <para>Check the<emphasis role="bold"> Preserve
+                Compression</emphasis> box to retain the file
+                compression.</para>
+              </listitem>
+
+              <listitem>
+                <para>Check the<emphasis role="bold"> Expire in
+                (days)</emphasis> box to enter a number of days before
+                automatically removing the file from the queue. If omitted,
```

Comment:
queue? it removes the file. Look at the other instances of this text in the manual. They should all be the same. Also, the presentation/format is different than the others. Look at Spray Fixed, for example, on page 80